src/OSPSuite.UI/Extensions/LayoutControlItemExtensions.cs

```diff
@@ -17,74 +17,78 @@ public static void InitializeAsHeader(this LayoutControlItem layoutControlItem,
          layoutControlItem.Text = text.FormatForLabel();
       }
 
-      public static void AdjustLargeButtonSize(this LayoutControlItem layoutControlItem)
+      public static void AdjustLargeButtonSize(this LayoutControlItem layoutControlItem, LayoutControl layoutControl)
       {
-         layoutControlItem.AdjustSize(UIConstants.Size.LARGE_BUTTON_WIDTH, UIConstants.Size.LARGE_BUTTON_HEIGHT);
+         layoutControlItem.AdjustSize(layoutControl, UIConstants.Size.LARGE_BUTTON_WIDTH, UIConstants.Size.LARGE_BUTTON_HEIGHT);
       }
 
-      public static void AdjustLongButtonSize(this LayoutControlItem layoutControlItem)
+      public static void AdjustLongButtonSize(this LayoutControlItem layoutControlItem, LayoutControl layoutControl)
       {
-         layoutControlItem.AdjustSize(UIConstants.Size.LARGE_BUTTON_WIDTH, UIConstants.Size.BUTTON_HEIGHT);
+         layoutControlItem.AdjustSize(layoutControl, UIConstants.Size.LARGE_BUTTON_WIDTH, UIConstants.Size.BUTTON_HEIGHT);
       }
 
-      public static void AdjustTallButtonSize(this LayoutControlItem layoutControlItem)
+      public static void AdjustTallButtonSize(this LayoutControlItem layoutControlItem, LayoutControl layoutControl)
       {
-         layoutControlItem.AdjustSize(UIConstants.Size.BUTTON_WIDTH, UIConstants.Size.LARGE_BUTTON_HEIGHT);
+         layoutControlItem.AdjustSize(layoutControl, UIConstants.Size.BUTTON_WIDTH, UIConstants.Size.LARGE_BUTTON_HEIGHT);
       }
 
       /// <summary>
       /// Sets the layout control item size to custom and define the min and max width
       /// </summary>
-      public static void AdjustSize(this LayoutControlItem layoutControlItem, int width, int height)
+      public static void AdjustSize(this LayoutControlItem layoutControlItem, LayoutControl layoutControl, int width, int height)
```

Comment:
Well we do not have access to the layoutControl from the layoutitem so I need to break this method and make it not compatible...oh well